affiliated/index.html

```diff
@@ -272,7 +234,7 @@ <h1 id="becoming-an-affiliated-package">Becoming an Affiliated Package<a class="
 	any related affiliated packages. If your package is determined to meet the above
 	standards, it will be accepted and added to the affiliated package registry.
 	Note however that if packages become unmaintained or do not meet the standards
-	anymore, they may be removed from the list of affiliated packages.</p>
+	anymore, they may be removed from the list of affiliated packages, as per pyOpenSci policy.</p>
```

Comment:
if you want you can link to this

```suggestion
	anymore, they may be removed from the list of affiliated packages, <a href="https://www.pyopensci.org/software-peer-review/our-process/policies.html#after-acceptance-package-ownership-and-maintenance">as per pyOpenSci policy.</a></p>
```